Automattic/woocommerce-payments
# Fix Incorrect Warning Notice for Puerto Rico and auto-select United States in onboarding

Fixes #8001

#### Changes proposed in this Pull Request









This PR introduces two changes:
- Remove the warning notice for Puerto Rico (by adding it as a supported country)
- Pre-select United States during onboarding
#### Testing instructions





* Get this PR locally, and run `npm run dev`
* Go to WCPay Dev and check the `Force the WCPay plugin to act as disconnected from the WCPay Server` checkbox
* Go to WooCommerce > Settings and chose a supported country (like Portugal)
* Go back to Payments and click Finish Setup
* Fill in your name and email and click Next
* In the company details, Portugal should be prefilled (making sure we are not breaking the normal flow)
* Leave the onboarding, and go back to WooCommerce > Settings. **Choose Puerto Rico as a country**
* Open http://localhost:8083, and go to `wp_options` table. Delete the `wcpay_onboarding_flow_state`. This will delete your progress from the last step, so we can start over
* Now head to Payments and make sure that the warning notice "WooPayments is not available in Puerto Rico" **isn't showing**. Click Finish Setup
* Fill in your name and email, and click Next
* In the company details page, make sure that the **United States is prefilled** in the country field.

-------------------


**Post merge**
